swagger/v2/post_people_responses.yaml

```diff
@@ -0,0 +1,52 @@
+'201':
+  type: object
+  required:
+  - data
+  properties:
+    data:
+      $ref: person.yaml#/Person
+'400':
```

Comment:
See this over and over. Is it possible to include `$ref: generic_errors.yaml` or something?

Reply:
In my next PR I'll do some work with swagger, let's have a chat about this, I can do it later.

Reply:
Sounds good dude!